### PR TITLE
Add variable_tags column to CollapsedVariables 

### LINKED ISSUE #180: Query for materialized view `collapsed_vars_mv` is likely wrong
This matview now contains rows with column `vars` with almost certainly undesirable values such as

```
lwe_thickness_of_precipitation_amountt: sum within months t: mean over years, ...
```

This is probably due to an unanticipated (but valid) change in the content of the `cell_method` column for some variables, which means the query for this view/matview is wrong.

At present, he conversion of each contributing `cell_method` column is expressed in SQLAlchemy by `func.regexp_replace(Variable.cell_method, "time: ", "_", "g")`. The intention of that replacement is apparently to create a conventional programming-language identifier. The obvious generalization would be to allow for any dimension preceding a colon (so, e.g., "t:"), and to replace all spaces with underscores. That generalization would still not handle all cases, but it would handle many more; it might also produce unfeasibly long identifiers.

Investigate, determine correct query, and create a migration to update the matview.

### PR DESCRIPTION
**Pull request by rod-glover**
Resolves part of #180

- Actually, the new column name is `unique_variable_tags`.
- Column `vars_ids` has also been added for possible later utility.
- The column `vars` has been preserved as-is because of external dependencies. However, its former use to distinguish climatological variables has been/will be replaced in https://github.com/pacificclimate/pdp_util/issues/50 with use of column `unique_variable_tags`.

### CODE REVIEW

File: pycds/orm/native_matviews/version_fecff1a73d7e.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# the connection between variables and histories. The main reason for its existence is
+# to enable the arrays of variable tags to be flattened and unique'd, which requires or
+# at least is most easily done with an intermediate stage created here. Other things are
+# placed here too, mainly for readability, but they could just as easily be computed
+# directly within the matview selectable.
+aggregated_vars = (
+    select(
+        VarsPerHistory.history_id.label("history_id"),
+        func.array_agg(VarsPerHistory.vars_id).label("vars_ids"),
+        func.array_agg(aggregate_order_by(variable_tags(Variable), Variable.id)).label(
```
Comment (Nospamas) [inline pycds/orm/native_matviews/version_fecff1a73d7e.py:41]: Unclear exactly what "all variable tags" is for as a new user, could be explained a little better.

File: tests/behavioural/conftest.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def env_config(schema_name):
 
 @pytest.fixture(scope="module")
 def target_revision():
-    return "bb2a222a1d4a"
+    return "fecff1a73d7e"
```
Comment (Nospamas) [inline tests/behavioural/conftest.py:102]: These hash targets are a little arcane for a new user, though easily explained. Quick comment might help point people in the right direction.

**Review by Nospamas — COMMENTED** [review]
Two notes above, discussed with Rod. Otherwise code looks clean :).

File: pycds/database.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 
 def check_migration_version(
-    executor, schema_name=get_schema_name(), version="bb2a222a1d4a"
+    executor, schema_name=get_schema_name(), version="fecff1a73d7e"
```
Comment (rod-glover) [inline pycds/database.py:8]: I wonder if this can and should be done with `pycds.alembic.info.get_head_revision()`

**Review by Nospamas — APPROVED** [review]
:+1:

**Pull request merged**